src/browser/provider/built-in/remote.js

```diff
@@ -13,7 +16,15 @@ export default {
 
         await this.waitForConnectionReady(browserId);
 
-        const localBrowserWindow = await findWindow(browserId);
+        let localBrowserWindow = null;
+
+        try {
+            localBrowserWindow = await findWindow(browserId);
+        }
+        catch (err) {
+            // NOTE: suppress the error, because finding of the local browser window was failed
```

Comment:
I really like it, but let's rephrase it a bit
```suggestion
            // NOTE: We can suppress the error here since we can just disable window manipulation functions
            // when the browser is truly remote and we cannot find a local window descriptor
```